t/exit.t

```diff
@@ -10,7 +10,7 @@ log_level('warn');
 repeat_each(120);
 #repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 5);
+plan tests => repeat_each() * (blocks() * 5) - (repeat_each() * 2);
```

Comment:
@thibaultcha Take a closer look at my form:

```perl
plan tests => repeat_each() * (blocks() * 5 - 2);
```

I believe you are misunderstanding it here.